docs/deployment/cloud-run.md

```diff
@@ -0,0 +1,108 @@
+# Deploy LogPaste to Google Cloud Run
+
+TODO: Finish this, still a work in progress.
+
+## Requirements
+
+* gcloud SDK
+
+## Set variables
+
+```bash
+GCP_PROJECT="your-gcp-project"    # Replace with your GCP project ID
+GCS_BUCKET="your-logpaste-bucket" # Replace with your bucket name
+GCP_REGION="us-east1"             # Replace with your desired GCP region
+```
+
+### Authenticate
+
+You need to authenticate gcloud and configure docker to push to Google Container Registry.
+
+```bash
+gcloud auth login && \
+  gcloud auth configure-docker
+```
+
+### Specify GCP Project
+
+```bash
+gcloud config set project "${GCP_PROJECT}"
+```
+
+### Enable services
+
+Your GCP project will need the [Cloud Run API](https://cloud.google.com/run/docs/reference/rest) enabled, so ensure your project enables this API:
+
+```bash
+gcloud services enable run.googleapis.com
+```
+
+### Create a Google Cloud Storage bucket
+
+If you haven't already created the GCS bucket to provide LogPaste's persistent storage, create it with the command below:
+
+```bash
+gsutil mb "gs://${GCS_BUCKET}"
+```
+
+## Push to Google Container Registry
+
+Cloud Run can't run Docker images from external image repositories, so next you'll copy the official LogPaste image to Google Container Registry:
+
+```bash
+LOGPASTE_VERSION="mtlynch/logpaste:latest"  # LogPaste version on DockerHub
+LOGPASTE_GCR_TAG="logpaste"                 # Change to whatever name you prefer
```

Comment:
this isn't technically a tag, I recommend using `LOGPASTE_GCR_IMAHE_NAME`

Reply:
Thanks, fixed!